model.py

```diff
@@ -8812,42 +8812,95 @@ class ExternalIntegration(Base):
     to a third-party API.
     """
 
-    # Utilities
+    # Possible types of ExternalIntegrations.
+    #
+    # These integrations are associated with external services such as
+    # Google Enterprise which authenticate library administrators.
+    ADMIN_AUTH_TYPE = 'admin_auth'
```

Comment:
Since we don't use most of these constants yet, this is mainly planning out what future use of ExternalIntegration will look like. I did this in this branch to see whether it was wise to remove the uniqueness constraint below.